calico-enterprise_versioned_docs/version-3.14/reference/resources/globalthreatfeed.mdx

```diff
@@ -204,9 +204,13 @@ expression, subject to the limitations mentioned below, that evaluates
 to a list of strings. For example: `$.` is valid for `["a", "b", "c"]`,
 and `$.a` is valid for `{"a": ["b", "c"]}`.
 
-> **LIMITATIONS**: No support for subexpressions and filters. Strings in
-> brackets must use double quotes. It cannot operate on JSON decoded
-> struct fields.
+:::caution
```

Comment:
@alexeymagdich-tigera This doesn't quite fit. I'll follow with https://tigera.atlassian.net/browse/DOCS-1196